src/ui/settingsPage/tableElement.js

```diff
@@ -0,0 +1,295 @@
+      border: 1px solid var(--border-color);
+      border-radius: 5px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .tableHolder table {
+      width: 100%;
+    }
+    h2 {
+      font-size: 38px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 40px; /* 105.263% */
+      margin-bottom: 18px;
+    }
+    p {
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 169.336%;
+      margin-bottom: 32px;
+    }
+    .tableHeader {
+      height: 56px;
+    }
+    .tableHeader th {
+      text-align: left;
+      padding-left: 16px;
+    }
+
+    .emptyState {
+      padding: 50px 10px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+    }
+    h3 {
+      font-size: 24px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 26px;
+      margin-bottom: 16px;
+    }
+    main p {
+      font-size: 14px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 21px;
+    }
+
+    .tableHolder th {
+      background-color: var(--panel-bg-color);
+    }
+    .tableHolder th button {
+      background-color: transparent;
+      border: none;
+      position: relative;
+    }
+    .tableHolder th button.sorted-down::after {
+      content: " ";
+      position: absolute;
+      top: -5px;
+      background-image: url("../../assets/img/arrow-toggle.svg");
+      background-position: center center;
+      background-repeat: no-repeat;
+      margin-inline-start: 8px;
+      pointer-events: none;
+      transform: rotate(0deg);
+      transition: transform 0.275s ease-in-out;
+      inline-size: 24px;
+      height: 24px;
+    }
+    .tableHolder th button.sorted-up::after {
+      content: " ";
+      position: absolute;
+      top: -5px;
+      background-image: url("../../assets/img/arrow-toggle.svg");
+      background-position: center center;
+      background-repeat: no-repeat;
+      margin-inline-start: 8px;
+      pointer-events: none;
+      transform: rotate(180deg);
+      transition: transform 0.275s ease-in-out;
+      inline-size: 24px;
+      height: 24px;
+    }
+    .tableHolder td {
+      padding: 16px;
+    }
+    .deleteBtn {
+      background-color: transparent;
+      color: transparent;
+      border: transparent;
+      background-image: url("../../assets/img/delete-gray.svg");
+      background-position: center center;
+      background-repeat: no-repeat;
+      transition: ease-in-out 0.3s;
+    }
+    .deleteBtn:hover {
+      filter: brightness(1.5);
+    }
+    td.delete {
+      width: 30px;
+    }
+    .row {
+      display: flex;
+      align-content: center;
+      align-items: center;
+    }
+  `;
+}
+customElements.define("mz-context-table", ContextTable);
+
+export const tableHeading = (
+  sortedBy = "",
+  acending = true,
```

Comment:
*ascending

Reply:
i really wish for a auto correct tool in an ide that does not suck haha.